### ISSUE TITLE
Wireframe - Feedback for Dashboard

### ISSUE THREAD

**Issue by briri** [CDLUC3/dmsp_frontend_prototype#102]

**Comment by fraserclark** [comment]
I believe this feedback has been applied and decision to go down the path of a list view and updated to start the figma prototype start page as the list view

https://www.figma.com/proto/RRtoRLyr5Spf0KiyRfEclt/DMP-Explain?node-id=886-846&node-type=frame&t=BiWgczUzRbAzAdyQ-0&scaling=min-zoom&content-scaling=fixed&page-id=177%3A504&starting-point-node-id=886%3A846&show-proto-sidebar=1

**Comment by briri** [comment]
@bofstein can we close this issue?

**Comment by bofstein** [comment]
Yeah I think we're good, looks great.  Could have minor changes down the line we can figure out with new tickets
